src/mobot_client/models/phone_numbers.py

```diff
@@ -0,0 +1,25 @@
+#  Copyright (c) 2021 MobileCoin. All rights reserved.
+
+#  Copyright (c) 2021 MobileCoin. All rights reserved.
+from phonenumbers import PhoneNumber
+from phonenumber_field.modelfields import PhoneNumberField as BasePhoneNumberField
+
+
+class PhoneNumberWithRFC3966(PhoneNumber):
+    @property
+    def country_code_string(self) -> str:
+        return f"+{self.country_code}"
+
+    @property
+    def rfc3966(self) -> str:
+        return f"+{self.country_code}{self.national_number}"
+
+    def __str__(self) -> str:
+        return f"+{self.country_code}{self.national_number}"
```

Comment:
`django-phonenumber` already handles this stuff https://github.com/stefanfoulis/django-phonenumber-field/blob/main/phonenumber_field/phonenumber.py#L36

Reply:
Yeah, I switched over to as_e164. Thanks!

Reply:
Yeah, I found the e164 stuff afterward.